doc/source/serve/deployment-graph.md

```diff
@@ -39,37 +44,38 @@ __[Full Ray Enhancement Proposal, REP-001: Serve Pipeline](https://github.com/ra
 
 ## Concepts
 
-- **Deployment**: Scalable, upgradeable group of actors managed by Ray Serve. __[See docs for detail](https://docs.ray.io/en/master/serve/core-apis.html#core-api-deployments)__
+- **Deployment**: Scalable, upgradeable group of actors and functions managed by Ray Serve. __[See docs for detail](https://docs.ray.io/en/master/serve/core-apis.html#core-api-deployments)__
 
 - **DeploymentNode**: Smallest unit in a graph, created by calling `.bind()` on a serve decorated class or function, backed by a Deployment.
 
 - **InputNode**: A special node that represents the input passed to a graph at runtime.
 
-- **Deployment Graph**: Collection of deployment nodes bound together to define an inference graph. The graph can be deployed behind an HTTP endpoint and reconfigured/scaled dynamically.
+- **Deployment Graph**: A collection of deployment nodes bound together to define an inference graph. The graph can be deployed behind an HTTP endpoint and reconfigured/scaled dynamically.
 
 +++
 
 ## Full End to End Example Walkthrough
 
-Let's put the concepts together and incrementally build a runnable DAG example highlighting the following features:
+Let's put the concepts together and incrementally build a runnable dynamic acyclic graph (DAG) example highlighting the 
+following features:
 
 ```{tip}
-At the end of this document we have the full and end to end executable implementation.
+At the end of this document, we'll have an end to end executable implementation of a deployment graph.
 ```
 
 - Building a graph:
     - A deployment node is created from `@serve.deployment` decorated function or class. 
     - You can construct different deployment nodes from same class or function.
-    - Deployment nodes can be used as input args in .bind() of other nodes in the DAG. 
+    - Deployment nodes can be used as input args in `.bind()` of other nodes in the DAG. 
     - Multiple nodes in the deployment graph naturally forms a DAG structure.
 - Accessing input:
     - Same input or output can be used in multiple nodes in the DAG.
     - Deployment nodes can access partial user input.
-- Dynamic control flow:
+- Dynamically controlling flow:
```

Comment:
nit: I think control flow is a programming language concept that is more accurate to stay as is

Reply:
No problem, make sense in that context. Will revert it back!